pkg/ssh/exec.go

```diff
@@ -184,6 +187,60 @@ More information is available here: https://okteto.com/docs/reference/manifest#r
 	return err
 }
 
+func waitForOtherCopyToFinish() {
+	wasRunning := false
+Loop:
+	for {
+		select {
+		case isRunning := <-isAnotherCopyRunning:
+			if !isRunning {
+				break Loop
+			} else {
+				wasRunning = true
+			}
+		default:
+			if !wasRunning {
+				break Loop
+			}
+		}
+	}
+}
+
+func injectLostCharacter(remoteStdin io.WriteCloser) {
+	select {
+	case char := <-lostCharacter:
+		remoteStdin.Write(char)
+	default:
+		break
+	}
+}
+
+func copyFromLocalToRemote(local io.Reader, remote io.WriteCloser, buf []byte) {
+	for {
+		nr, er := local.Read(buf)
```

Comment:
could we do a `select`here between `Read` and the `lostCharacter` channel?

Reply:
no, I tried that but it will get blocked anyways on the read until you press a key